backend/ibutsu_server/filters.py

```diff
@@ -99,7 +102,17 @@ def convert_filter(filter_string, model):
         return None
     field = match.group(1)
     oper = match.group(2)
-    value = match.group(3).strip('"')
+    value = match.group(3)
+    is_string = False
+    is_version = False
+    match = VERSION_RE.match(value)
+    if match:
+        value = match.group(1)
+        is_version = True
+    match = STRING_RE.match(value)
+    if match:
+        value = match.group(1)
+        is_string = True
```

Comment:
Changes here were part of a debugging session with @rsnyman where we are trying to fix a problem with comparing BaseOS. Still need to revisit.